sdk/provisioning/Azure.Provisioning.CloudMachine/src/Client/MessagingServices.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Messaging.ServiceBus;
+
+namespace Azure.CloudMachine;
+
+public static class MessagingServices
+{
+    public static void Send(this CloudMachineClient cm, object serializable)
+    {
+        ServiceBusSender sender = cm.ClientCache.Get<ServiceBusSender>("cm_default_topic", () =>
+        {
+            ServiceBusClient sb = new(cm.Properties.ServiceBusNamespace, cm.Credential);
```

Comment:
Should you be caching the `SBClient` separately first?

Reply:
Yeah, I was thinking about it, but I am not sure how to design good cache APIs like that.

Reply:
I did double caching. A bit complex, but works.